packages/expo-battery/src/ExpoBattery.web.ts

```diff
@@ -1 +1,97 @@
-export default {};
+import { EventEmitter } from '@unimodules/core';
+import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
```

Comment:
this dependency on fbjs is going to cause problems in the future. fbjs is not meant to be a dependency. better to extract this into another library like 'expo-web-environment'

Reply:
react-native-web uses this to detect SSR.

Reply:
i'm aware, it's that I don't think we want to recommend this. fbjs is not meant to have a public api.